include/deployer.bash

```diff
@@ -76,10 +76,15 @@ cbd-version() {
 }
 
 cbd-update() {
-    declare desc="Binary selfupdater. Either latest github release (default), or specific branch from CircleCI"
+    declare desc="Binary selfupdater. Either latest github release"
 
     if [[ "$1" ]]; then
-        cbd-update-snap $1
+        # cbd-update-snap $1
+        warn "!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+        warn "This version is unsupported"
```

Comment:
otherwise it might suggest:
- users try `cbd update 1.6-rc`: this version is not supported
- ooo lets try an other version: `cbd update 1.7-rc` : this version is not supported
- and he can try any parameter he will get a "this version is not supported" but wont realize, that "no parameter is supported"